packages/angular-cli/blueprints/ng2/files/__path__/tsconfig.json

```diff
@@ -12,6 +12,11 @@
     "target": "es5",
     "typeRoots": [
       "../node_modules/@types"
-    ]
+    ],
+    "baseUrl": ".",
+    "paths": {
+      "@<%= prefix %>/*": ["app/*"],
+      "app/*": ["app/*"]
```

Comment:
Actually, thinking about it a little bit.  It might be good to just add support for `paths` and add defaults in a follow-up PR.  Many people that want this will most likely customize anyway.